app/src/gui/components/record/relationships/create_links/index.tsx

```diff
@@ -79,29 +75,23 @@ export default function CreateLinkComponent(
               />
             )}
 
-            {props.allowLinkToExisting !== false && (
```

Comment:
Did you mean to remove this check for the allowLinkToExisting - a new prop from the configurable link to existing?